.github/workflows/ci.yml

```
@@ -0,0 +1,392 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      # Feature branches beginning with "feature."
+      - feature.*
+    tags:
+      - '**'
+  pull_request:
+
+jobs:
+  sass_spec:
+    runs-on: ubuntu-latest
+
+    name: "sass-spec | Dart ${{ matrix.dart_channel }} | ${{ matrix.async_label }}"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        dart_channel: [stable, dev]
+        async_label: [synchronous]
+        async_args: ['']
+        include:
+          - dart_channel: stable
+            async_label: asynchronous
+            async_args: '--cmd-args --async'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Dart
+        uses: cedx/setup-dart@v2
+        with:
+          release-channel: "${{ matrix.dart_channel }}"
+
+      - name: Install app dependencies
+        run: dart pub get
+
+      # Setup the environment to run the sass-spec runner which is written in Ruby
+      - name: Setup sass-spec
+        # TODO replace the spec-ref script
+        run: |
+          export sass_spec_ref=`tool/github-action/sass-spec-ref.sh`
+          git init sass-spec
+          git -C sass-spec fetch git://github.com/sass/sass-spec "$sass_spec_ref" --depth 1
+          git -C sass-spec checkout FETCH_HEAD
+        env:
+          PR_BRANCH: "${{ github.base_ref }}"
+          CURRENT_BRANCH: "${{ github.ref }}"
+          PR_BODY: "${{ github.event.pull_request.body }}"
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+          working-directory: sass-spec
+          bundler-cache: true
+
+      - name: Run specs
+        run: bundle exec sass-spec.rb --dart .. $extra_args
+        working-directory: ./sass-spec
+        env:
+          extra_args: "${{ matrix.async_args }}"
+
+  dart_tests:
+    runs-on: "${{ matrix.os }}"
+
+    name: "Dart tests | Dart ${{ matrix.dart_channel }} | ${{ matrix.os }}"
+
+    strategy:
+      fail-fast: false
+
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        dart_channel: [stable]
+        include:
+          - os: ubuntu-latest
+            dart_channel: dev
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Dart
+        uses: cedx/setup-dart@v2
+        with:
+          release-channel: "${{ matrix.dart_channel }}"
+
+      - name: Install app dependencies
+        run: dart pub get
+
+      - name: Build
+        run: dart pub run grinder pkg-standalone-dev
+
+      - name: Run tests
+        run: dart pub run test -p vm -x node
```

```suggestion
        run: dart pub run test -p vm -x node -r expanded
```

This will cause the test runner to only print output when a test passes or fails, rather than printing a line every second.